examples/OECC/config/customer.p

```diff
@@ -0,0 +1,33 @@
+USING Progress.Json.ObjectModel.JsonObject.
```

Comment:
Should include a header for the program to describe purpose and/or where the file should be placed. Also, if this is expected to be found via the CompatibilityHandler in a PAS instance that would provide context in case the user can't figure out why they're unable to find and run the file.

Reply:
Added header for the program.